Docs 9568 review concept and task topics esb (#104)
mulesoft/docs-partner-manager: modules/ROOT/pages/delete-endpoints.adoc

= Delete Endpoints

> You can delete endpoints that are not used in message flows from the Host or Partner page:

Reviewer: ```suggestion
If they are not being used, you can delete endpoints from the Host or Partner page:
```
NOTE that "Host or Partner page" are capitalized here and not at Line 3, and that, again, it is unclear whether this means...:

Either the host page or the partner page
Either the host (not a page) or the partner page

. From Anypoint Partner Manager, select the *<host name>* or *Partners* from the left navigation menu.
. Locate the endpoint that you want to delete.
+

> Note that you can the list for any endpoints that don’t have an associated message flow. Endpoints that are not associated with a message flow are indicated by a `0` in the *Message Flows* column.

Reviewer: ```suggestion
Note that you can search for any endpoints that don’t have an associated message flow by looking for `0` in the *Message Flows* column.
```

Reviewer: Don't we have to mention hovering or the associated popup icon?

Author: Done!

+
. Hover over the endpoint name until you see the garbage can icon.

> . Select the trashcan icon and click *Delete*.

Reviewer: ditto ^ : the trashcan icon where?

Reviewer: ```suggestion
. Hover over the endpoint name, select the trashcan icon that appears, and click *Delete*.
```

Author: Done
